.vale.ini

```diff
@@ -1,6 +1,30 @@
-StylesPath = ".github/styles"
-Vocab = "Docs"
+# For more information, see ``.github/vale/README.rst``
+#
+# vale-action (https://github.com/errata-ai/vale-action) recommends
+# keeping the vale styles in the `.github` directory.
+# Since we have a README, styles, a dictionary, and some tests, we are
+# keeping related directories files in `.github/vale`
+#
```

Comment:
Remove this second paragraph below? I think "it's all working" now, right?

Reply:
Oh, yes, thanks.